### PR TITLE
Feature/micronaut port

### PR DESCRIPTION
**Pull request by 0xCA5A**

### CODE REVIEW

File: src/main/java/ch/jbert/JbertApplication.java [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 import io.micronaut.runtime.Micronaut;
 
-public class Application {
-
+public class JbertApplication {
     public static void main(String[] args) {
-        Micronaut.run(Application.class);
+        Micronaut.run(JbertApplication.class);
```
Comment (rake5k) [inline src/main/java/ch/jbert/JbertApplication.java]: Doesn't `Application` suffice?

File: build.gradle [hunk 2]
```diff
@@ -38,11 +41,14 @@ dependencies {
     testImplementation "org.junit.jupiter:junit-jupiter-api"
     testImplementation "io.micronaut.test:micronaut-test-junit5"
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine"
+
+    // Sub-Project dependencies
+    compile project(':core')
```
Comment (rake5k) [inline build.gradle:66]: I'm not a huge fan of having a `core` module. This usually ends up having the whole application in it, without clear distinction between logical units and possibly a big dependency mess. Is it already possible to split the core into independent modules, like `rfid`, `gpio`, `shared` (or `common` for shared classes)?

File: src/main/resources/logback.xml [hunk 3]
```diff
@@ -9,7 +9,21 @@
...
+    <appender name="ASYNCSTDOUT" class="ch.qos.logback.classic.AsyncAppender">
+        <appender-ref ref="STDOUT"/>
+    </appender>
+
+    <logger name="ch.jbert" level="DEBUG"/>
+    <logger name="ch.jbert.event" level="DEBUG"/>
+    <logger name="ch.jbert.gpio" level="DEBUG"/>
+    <logger name="ch.jbert.mpd" level="DEBUG"/>
+    <logger name="ch.jbert.rfid" level="DEBUG"/>
+    <logger name="ch.jbert.util" level="DEBUG"/>
```
Comment (rake5k) [inline src/main/resources/logback.xml:21]: If all the sub-packages have the same log level, declaration of just the `ch.jbert` logger should suffice.

File: README.md [hunk 4]
```diff
@@ -90,11 +69,8 @@ sam@guido:~/projects/git/jbert$ rsync --chmod=664 --chown=mpd:audio -avh music p
 ### Monitor Target JVM
 Due to the fact that the Raspberry Pi Zero hardware is not the most powerful one it might be good to understand
 what is going on on the target JVM. This can be done using VisualVM.
- * JMX is enabled in the javaOptions defined in `build.sbt`
+ * JMX is enabled in the javaOptions defined in `build.gradel`
```
Comment (rake5k) [inline README.md]: Typo: `gradel`

File: build.gradle (hunk 2 above)
Comment (0xCA5A) [reply build.gradle:66]: 'core' is in this case more 'backend' - no dependencies to micronaut.
What do you suggest? Should we habe sub projects for all packages like gpio / rfid?

File: src/main/resources/logback.xml (hunk 3 above)
Comment (0xCA5A) [reply src/main/resources/logback.xml:21]: Right. But it sucks then to change it on the target for one package...

File: src/main/java/ch/jbert/JbertApplication.java (hunk 1 above)
Comment (0xCA5A) [reply src/main/java/ch/jbert/JbertApplication.java]: fixed

File: README.md (hunk 4 above)
Comment (0xCA5A) [reply README.md]: fixed

File: build.gradle (hunk 2 above)
Comment (rake5k) [reply build.gradle:66]: At least I would consider to separate the mpd- from the hardware-stuff - but this can be done in a separate step, as well.

**Review by rake5k — APPROVED** [review]

File: build.gradle (hunk 2 above)
Comment (0xCA5A) [reply build.gradle:66]: ill open a separate pr for this

**Pull request merged**